packages/prompts/src/index.ts

```diff
@@ -409,6 +414,27 @@ export const outro = (message = '') => {
 	process.stdout.write(`${color.gray(S_BAR)}\n${color.gray(S_BAR_END)}  ${message}\n\n`);
 };
 
+export type LogMessageOptions = {
+	symbol?: string;
+};
+export const log = {
+	message: (message = '', { symbol = color.gray(S_BAR) }: LogMessageOptions = {}) => {
+		process.stdout.write(`${color.gray(S_BAR)}\n${symbol}  ${message}\n`);
```

Comment:
Should probably handle multiline text here

```suggestion
		const lines = message.split('\n');
		process.stdout.write(`${color.gray(S_BAR)}\n${lines.map((ln, i) => `${i === 0 ? symbol : color.gray(S_BAR)}  ${ln}`).join('\n')}\n`);
```

Reply:
Good call!

Alternative implementation:

```ts
const [firstLine, ...lines] = message.split('\n');
process.stdout.write(`${color.gray(S_BAR)}\n${symbol}  ${firstLine}\n${lines.map((ln) => `${color.gray(S_BAR)}  ${ln}`).join('\n')}\n`);
```

Reply:
Yep that's cleaner!